### DISCUSSION TITLE
Add page.NavigationRail

### DISCUSSION THREAD

**Discussion by thiagovianav** [flet-dev/flet#1607] [Q&A]
I really like how NavigationRail looks so beautiful but I can't use it well because this needed to be placed in a Row or column and it affects negatively my program because some things like scrollbar :/

**Comment by FeodorFitsner** [comment]
NavigationRail is not the same as `app_bar` or `bottom_navigation` - it's an independent control, not a part of "scaffold". You can have multiple navigation rails on the same page.
If you have troubles with layout with navigation rail feel free to post your code sample to get an advice.
